src/current/cockroachcloud/advanced-cluster-management.md

```diff
@@ -98,18 +98,18 @@ You can add up to nine regions at a time through the Console. See [Planning your
 
 1. On the **Regions** page, click **Add a region**.
 
-    If you have a GCP cluster with [VPC peering]({% link cockroachcloud/network-authorization.md %}) enabled, the IP range will be automatically populated for added regions.
+    For GCP, Each region consumes a `/19` CIDR range from the CIDR range configured during cluster creation. If you add a region a GCP cluster with [VPC peering]({% link cockroachcloud/network-authorization.md %}) enabled, the region's IP range will be automatically assigned from the cluster's CIDR range.
```

Comment:
> If you add a region `to` a GCP

Missing a word I think.

/nit
> the region's IP range will be automatically assigned from the cluster's CIDR range.

This work flow is true if they have VPC peering enabled or not

Reply:
Thanks, I was trying to copy some of the detail from the cluster-create page, which says it's populated automatically if you add additional regions. Does this work?